students/ericstreit/lessons/lesson04/assignment/create_customers.py

```diff
@@ -0,0 +1,54 @@
+"""
+#Lesson03
+#Customer Database Exercise ##
+"""
+
+# !/usr/bin/env python3
+# import stuff here
+from db_model import *
+import logging
+logging.basicConfig(level=logging.DEBUG)
+# global variables here
+
+# define function
+def create_customers(first__name, last__name, home__address, phone__number, email__address,
+                     account__status, credit__limit):
+    """
+    Description of function
+
+    :param arg1: The first very important parameter. And a bit about
+                 what it means.
+    :param arg2: The second very important parameter. And now some
+                 description of how this is used
+    etc
+    """
```

Comment:
Very nice doc string. ;) Do get it in the habit of writing good doc strings; they are very useful.